diff --git a/node_modules/react-native/.DS_Store b/node_modules/react-native/.DS_Store
new file mode 100644
index 0000000..2aa0eb0
Binary files /dev/null and b/node_modules/react-native/.DS_Store differ
diff --git a/node_modules/react-native/scripts/cocoapods/spm.rb b/node_modules/react-native/scripts/cocoapods/spm.rb
index 0b13207..9358a0f 100644
--- a/node_modules/react-native/scripts/cocoapods/spm.rb
+++ b/node_modules/react-native/scripts/cocoapods/spm.rb
@@ -31,6 +31,7 @@ class SPMManager
         )
         log " Adding workaround for Swift package not found issue"
         target = project.targets.find { |t| t.name == pod_name}
+        next if target.nil?
         target.build_configurations.each do |config|
           target.build_settings(config.name)['SWIFT_INCLUDE_PATHS'] ||= ['$(inherited)']
           search_path = '${SYMROOT}/${CONFIGURATION}${EFFECTIVE_PLATFORM_NAME}/'
@@ -66,6 +67,7 @@ class SPMManager
   end
 
   def add_spm_to_target(project, target, url, requirement, products)
+    return if target.nil?
     pkg_class = Xcodeproj::Project::Object::XCRemoteSwiftPackageReference
     ref_class = Xcodeproj::Project::Object::XCSwiftPackageProductDependency
     pkg = project.root_object.package_references.find { |p| p.class == pkg_class && p.repositoryURL == url }
